test/client/fixtures/utils/dom-test.js

```diff
@@ -364,6 +364,80 @@ test('isHammerheadAttr', function () {
     ok(domUtils.isHammerheadAttr('src-hammerhead-stored-value'));
 });
 
+test('isContentEditableElement', function () {
+    notOk(domUtils.isContentEditableElement(null));
+
+    document.designMode = 'on';
+    ok(domUtils.isContentEditableElement(document));
+    ok(domUtils.isContentEditableElement(document.body));
+    document.designMode = 'off';
+
+    // isRenderedNode
+    /* eslint-disable no-empty */
+    try {
+        var processingInstruction = document.createProcessingInstruction('xml-stylesheet', 'href="mycss.css" type="text/css"');
```

Comment:
Does `try...catch` connect with Android 5.1?
Add comment why we need try/catch there.

Reply:
in IE and Android `document.createProcessingInstruction` raises an error, ok I''ll add comment

Reply:
https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/fixtures/utils/dom-test.js#L131-L132

Reply:
ok, thanks